packages/eas-build-job/src/job.ts

```diff
@@ -32,15 +39,25 @@ export function sanitizeJob(rawJob: object, { sdkVersion }: { sdkVersion?: strin
   }
 }
 
+function setAndroidBuilderImage(job: Job, reactNativeVersion?: string): void {
+  if (!(!job.builderEnvironment?.image && reactNativeVersion)) {
```

Comment:
I'd invert the condition -> `job.builderEnvironment?.image || reactNativeVersion`. IMO it's hard to read the original if.

Reply:
ios part have condition structured in a similar way do you want me to change it there?